lib/main.c

```diff
@@ -0,0 +1,7 @@
+#include "marshal.h"
+
+int
+main()
+{
+        return 0;
+}
```

Comment:
This is our local test file to test marshal.h APIs. Grace will work as per travis framework. I will remove these.